src/styles/forms/_general.scss

```diff
@@ -1,19 +1,19 @@
 .form-group {
-  $borderColor: var(--KInputBorder, var(--grey-300, color(grey--grey-300)));
+  $borderColor: var(--KInputBorder, var(--grey-300, #e7e7ec));
   display: block;
-  margin-bottom: var(--lg, spacing(lg));
+  margin-bottom: var(--lg, 24px);
```

Comment:
do we need this `--lg` property? Doesn't seem standard and I don't see that we document it anywhere. Same for `--xl` etc.